### PR TITLE
Remove blob

### LINKED ISSUE #497: As a user, I want to receive metadata only in the API responses (no binary blobs)
### Checked for duplicates

No - I haven't checked

### 🧑‍🔬 User Persona(s)

API users

### 💪 Motivation

...so that the response is not overloaded with useless data and the performance are not negatively impacted by this data.

### 📖 Additional Details

The opensearch database contains blobs of XML or JSON PDS4 labels. 

The user must not get these blobs in the defaults API responses and should only be able to get them decoded when he/she specifically asks for the PDS4 Json or XML response formats.

### Acceptance Criteria

**Given** 
**When I perform** 
**Then I expect** 

### ⚙️ Engineering Details

_No response_

### 🎉 I&T

_No response_

### LINKED ISSUE #514: Manage renewal of AWS credentials
### 💡 Description

AWS credentials allocated to the API expire every 5 hours. Some code need to be written to renew them.

### ⚔️ Parent Epic / Related Tickets

_No response_

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
Removes the blob from the application/json and kvp responses.

## ⚙️ Test Data and/or Report

Test with postman, see PR : https://github.com/NASA-PDS/registry/pull/301

## ♻️ Related Issues

Fixes https://github.com/NASA-PDS/registry-api/issues/497
Fixes https://github.com/NASA-PDS/registry-api/issues/514

### CODE REVIEW

File: terraform/ecs.tf
Comment (alexdunnjpl) [inline terraform/ecs.tf:1]: Were these changes an intentional part of this PR?

File: service/src/main/java/gov/nasa/pds/api/registry/util/AWSCredentialsFetcher.java
Comment (alexdunnjpl) [inline service/src/main/java/gov/nasa/pds/api/registry/util/AWSCredentialsFetcher.java:1]: Were these changes an intentional part of this PR?

File: service/src/main/java/gov/nasa/pds/api/registry/model/api_responses/ProductBusinessLogicImpl.java [hunk 1]
```diff
@@ -93,11 +108,15 @@ public static Map<String, List<String>> getFilteredProperties(Map<String, Object
...
     if ((included_fields == null) || (included_fields.size() == 0)) {
       String apiProperty;
+      log.debug("Excluded fields are " + excluded_fields);
       for (Map.Entry<String, Object> entry : sourceAsMap.entrySet()) {
         try {
           apiProperty = SearchUtil.openPropertyToJsonProperty(entry.getKey());
-          if ((excluded_fields == null) || !excluded_fields.contains(apiProperty))
+          log.debug("see if property " + apiProperty + "should be added");
+          if ((excluded_fields == null) || !excluded_fields.contains(apiProperty)) {
+            log.debug("confirmed, the property is being added");
```
Comment (alexdunnjpl) [inline service/src/main/java/gov/nasa/pds/api/registry/model/api_responses/ProductBusinessLogicImpl.java]: Are these lines intended to remain in the codebase, or were they temporary dev logs?

**Review by alexdunnjpl — APPROVED** [review]
LGTM, with some questions (see comments)

File: service/src/main/java/gov/nasa/pds/api/registry/model/api_responses/ProductBusinessLogicImpl.java (hunk 1 above)
Comment (tloubrieu-jpl) [reply service/src/main/java/gov/nasa/pds/api/registry/model/api_responses/ProductBusinessLogicImpl.java]: Right temporary logs. I can remove them.

File: terraform/ecs.tf
Comment (tloubrieu-jpl) [reply terraform/ecs.tf:1]: They are also part of another PR https://github.com/NASA-PDS/registry-api/pull/482 which was not approved/merged. SoI embedded it here.

**Pull request merged**